### PR TITLE
Ros 5

### PR DESCRIPTION
**Pull request by canyon289**
Basically done except the the default histogram from ArviZ at the end doesn't look great

### CODE REVIEW

**Comment by aloctavodia** [comment]
The histogram on arviz assumes by default that you want integer bins. It you want to override this pass hist_kwargs={"bins":"auto" } or something similar

**Comment by canyon289** [comment]
Oh cool will try that. Thank you!

**Comment by canyon289** [comment]
It didnt work out so well. I got some exception so just used matplotlib directly

File: ROS/Chapter_5.ipynb [hunk 1]
```diff
@@ -0,0 +1,505 @@
+{
```
Comment (aloctavodia) [inline ROS/Chapter_5.ipynb:1]: <p>reorder like we do in pymc resources?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/Bambi_resources/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (aloctavodia) [inline ROS/Chapter_5.ipynb:1]: <p>preallocating the rvs should make this much more faster</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/Bambi_resources/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (aloctavodia) [inline ROS/Chapter_5.ipynb:1]: <p>use "constrained_layout=True"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/Bambi_resources/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (aloctavodia) [inline ROS/Chapter_5.ipynb:1]: <p>add semicolon to the last line</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/Bambi_resources/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (aloctavodia) [inline ROS/Chapter_5.ipynb:1]: <p>also semicolon</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/Bambi_resources/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Comment (aloctavodia) [inline ROS/Chapter_5.ipynb:1]: <p>I will fix this on the ArviZ's side</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/Bambi_resources/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Comment (canyon289) [reply ROS/Chapter_5.ipynb:1]: <p>What does preallocating rvs mean here?</p>

Comment (canyon289) [reply ROS/Chapter_5.ipynb:1]: <p>This constrained layout does not look very good... </p>

Comment (canyon289) [reply ROS/Chapter_5.ipynb:1]: image

Comment (canyon289) [reply ROS/Chapter_5.ipynb:1]: Nevermind fixed it with some line breaks 
image

**Comment by canyon289** [comment]
Merging to start split into individual examples

**Pull request merged**